stand/lua/config.lua

```diff
@@ -657,12 +677,40 @@ function config.readConf(file, loaded_files)
 
 	if load_conf_dirs then
 		local loader_conf_dirs = getEnv("loader_conf_dirs")
+
+		-- If product_vars is set, it must be a list of environment variable names
+		-- to walk through to guess product information. The order matters as
+		-- reading a config files override the previously defined values.
+		--
+		-- If product information can be guessed, for each product information
+		-- found, also read config files found in /boot/loader.conf.d/PRODUCT/.
+		local product_vars = getEnv("product_vars")
+		if product_vars ~= nil then
```

Comment:
At some point I think I'd like to do a sweep of all of the lua in the tree and convert explicit comparisons against `nil` to `foo` or `not foo` as appropriate... probably 99% of occurrences today can be converted because they don't need to care about `nil` vs `false`.